Misc/NEWS.d/next/Documentation/2018-06-03-22-24-22.bpo-33766.0cYkXu.rst

```diff
@@ -0,0 +1,2 @@
+Document an edge case in the parser involving a missing newline at the end
```

Comment:
We don't require NEWS entries for small documentation patches, so I think this can be removed.

Reply:
Removed.